Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -76,13 +76,15 @@ import qualified Storage.Queries.SearchRequestForDriver as QSearchReqForDriver
 import qualified Storage.Queries.Vehicle as QVehicle
 import qualified Tools.Auth as Auth
 import Tools.Error
+import Storage.CachedQueries.TransporterConfig
 
 -- FIXME: not tested yet because of no onboarding test data
 driverDocumentsInfo :: ShortId DM.Merchant -> Flow Common.DriverDocumentsInfoRes
 driverDocumentsInfo merchantShortId = do
   merchant <- findMerchantByShortId merchantShortId
   now <- getCurrentTime
-  onboardingTryLimit <- asks (.driverOnboardingConfigs.onboardingTryLimit)
+  transporterConfig <- findByMerchantId merchant.id >>= fromMaybeM (MerchantServiceUsageConfigNotFound merchant.id.getId)
```

Comment:
You may use TC error from this commit of shared-kernel: `53e9f01c07efcd6f603015072624593f1faec90f `

Reply:
can you  merge that PR ?

Reply:
You can
1. Checkout to that branch locally
2. Point your shared kernel to your local repo